dev_info/build.html

```diff
@@ -0,0 +1,31 @@
+﻿<!DOCTYPE html>
+
+<link rel="stylesheet" href="../locales/help_enu/styles.css" type="text/css" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta charset="utf-8" />
+<title>Building GEDKeeper</title>
+</head>
+<body>
+<h1>How to build GEDKeeper</h1>
+<h2>Microsoft Windows</h2>
+<h3>Dependencies</h3>
+<p>The following is a list of required software to build and test:
+<ul>
+<li>Microsoft <a href="https://dotnet.microsoft.com/">.NET Framework</a> 4.0.30319</li>
+<li>Microsoft <a href="https://visualstudio.microsoft.com/">Visual Studio</a> 2017 or later</li>
+<li>Nullsoft Scriptable Install System (<a href="https://sourceforge.net/projects/nsis/">NSIS</a>) 3.04</li>
+<li><a href="https://sourceforge.net/projects/sevenzip/">7-zip</a></li>
```

Comment:
```sh
$ find . -type f \! -name *.resx -exec grep -Hn 7z \{\} \; | grep -v [Bb]inary
./deploy/gk2_win_portable.cmd:48:rem "c:\Program Files\7-zip\7z.exe" a -tzip -mx5 -scsWIN %zip_fn% @%lstfile% > %log_fn%
./deploy/gk2_win_portable.cmd:49:"c:\Program Files\7-zip\7z.exe" a -tzip -mx9 -scsWIN %zip_fn% @%lstfile%
./make_gk2_mswin_installer.cmd:16:"c:\Program Files\7-zip\7z.exe" a -tzip -mx5 -scsWIN .\deploy\gedkeeper_2.16.2_win.zip .\deploy\gedkeeper_2.16.2_winsetup.exe
```

I believe, both \`\`Nullsoft scriptable install system'' and \`\`7-zip'' are used for "packaging".  They aren't dependencies for **build**.

Reply:
Yes

Reply:
Nomenclature semantics, but certainly "packaging" works.

Reply:
Fixed.

Reply:
76f7917f5b08bb91ad2a1e627d8d8dddc8572666 is looked good to me.  Thanks!